### PR TITLE
NOISSUE-Update initialization of instanceID

### PR DESCRIPTION
**Pull request by ianmuchyri**
### What does this do?
removes unnecessary `instanceID` initialization and instead uses `cfg.InstanceID` directly.
### List any changes that modify/break current functionality
none
### Have you included tests for your changes?
no
### Did you document any new/modified functionality?
no

### CODE REVIEW

File: cmd/bootstrap/main.go [hunk 1]
```diff
@@ -73,10 +73,8 @@ func main() {
 	var exitCode int
 	defer mflog.ExitWithError(&exitCode)
 
-	instanceID := cfg.InstanceID
-	if instanceID == "" {
-		instanceID, err = uuid.New().ID()
-		if err != nil {
+	if cfg.InstanceID == "" {
+		if cfg.InstanceID, err = uuid.New().ID(); err != nil {
 			logger.Error(fmt.Sprintf("Failed to generate instanceID: %s", err))
```
Comment (rodneyosodo) [inline cmd/bootstrap/main.go]:
```suggestion
			logger.Error(fmt.Sprintf("failed to generate instanceID: %s", err))
```

Same as others

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Also rename this https://github.com/ianmuchyri/mainflux/blob/e92ca98508dae7eb65da4fa0a09da8b54e06933f/internal/clients/jaeger/provider.go#L41 as depicted by opentelemetry quidelines https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/host.md

**Review by rodneyosodo — APPROVED** [review]
LGTM

**Review by drasko — APPROVED** [review]
LGTM

**Pull request merged**